streaming/rrdpush.h

```diff
@@ -80,6 +80,7 @@ struct receiver_state {
     int update_every;
     uint32_t stream_version;
     time_t last_msg_t;
+    time_t gap_start, gap_end;
```

Comment:
Shouldn’t this be `size_t` too then?

Reply:
There is a strong argument for that. I'm going to do another sweep to pick one type for all of them - probably size_t to make it more portable.